src/Employment/WorkContractRequest_v1.0.py

```diff
@@ -0,0 +1,340 @@
+import datetime
+from datetime import date
+from enum import Enum
+from typing import List, Optional
+
+from definition_tooling.converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class WorkContractRequest(CamelCaseModel):
+    end_user_authentication_token: str = Field(
```

Comment:
Why is this here still after this comment?
<img width="529" alt="image" src="https://github.com/Virtual-Finland/definitions/assets/6637576/1bd3b1db-6095-4e54-9777-6646826913a8">
So the only input is the authentication token (that's gets enabled when you mark the definition with `requires_authorization=True,`, as is done, and there's not supposed to be any input field in the request.